### DISCUSSION TITLE
Add a tooltip for the column title and/or description field

### DISCUSSION THREAD

**Discussion by teranex** [inovex/scrumlr.io#4276] [Proposals]
We sometimes use longer column titles. Depending on the browser-width they are only shown partially followed by an ellipsis. As far as I can see there is in that case no way to see the entire column title. This could be solved by a simple tooltip on the column title. Alternatively allowing for a description field could also solve this issue, as we can then use shorter column titles and add some guidelines about that column in the description field.

**Comment by Schwehn42** [comment]
Hi Jeroen, great idea!  
I've just implemented the tooltip (https://github.com/inovex/scrumlr.io/pull/4279), should be coming with next release very soon.  
As for the description, this is also planned and is going to be added in context of the new template/session feature, which is also coming soon. :)
Hope that helps!

**Comment by teranex** [comment]
@Schwehn42 Awesome! Thanks a lot :)

**Comment by SelinaBuff** [comment]
I have also just created a [ticket for a description/subtitle field](https://github.com/inovex/scrumlr.io/issues/4280) in our backlog. This issue is expected to be addressed in the next sprint change. Thank you for the input @teranex! :)
